File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/VersionNotFoundException.java
package com.dogGetDrunk.meetjyou.common.exception.business;

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode;

public class VersionNotFoundException extends NotFoundException {

    private String version;

    public VersionNotFoundException(String version) {
        super(version, ErrorCode.VERSION_NOT_FOUND);
    }
}

File: src/main/java/com/dogGetDrunk/meetjyou/common/exception/business/notFound/VersionNotFoundException.kt
package com.dogGetDrunk.meetjyou.common.exception.business.notFound

import com.dogGetDrunk.meetjyou.common.exception.ErrorCode

class VersionNotFoundException(
    value: String,
    message: String? = null,
) : NotFoundException(ErrorCode.VERSION_NOT_FOUND, value, message)
